pkg/plugin/probes-handler/probes_handler.go

```diff
@@ -13,25 +15,36 @@ type Probe struct {
 }
 
 // NewProbesHandler registers liveliness and readinesss probes for /version endpoint
-func NewProbesHandler() http.Handler {
+func NewProbesHandler(log *logrus.Entry) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+
+		serverError := func(action string, err error) {
+			log.WithError(err).Errorf("Error %s.", action)
```

Comment:
Maybe some additional context in the message would help. eg: `Probe handler failed when`